src/eftest/runner.clj

```diff
@@ -82,13 +82,14 @@
                     (defaults to eftest.report.progress/report)"
   ([vars] (run-tests vars {}))
   ([vars opts]
-   (let [start-time (System/nanoTime)]
+   (let [start-time (System/nanoTime)
+         report     (:report opts progress/report)]
```

Comment:
This change can be reverted, I think, since you've just moved the value from the `binding` to the `let`.